### PR TITLE
[WIP] Frontend 

### PR DESCRIPTION
**Pull request by mpatsiou**
-Backround photo
-Sign in form
-Style for the form
Screenshot from 2020-04-12 20-38-35
Screenshot from 2020-04-13 00-38-44

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+</<!DOCTYPE html>
+<html lang="en" dir="ltr">
```
Comment (pkakelas) [inline index.html:2]: Why do you have this `dir` attirbute?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css "href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="style.css" />
```
Comment (pkakelas) [inline index.html:6]: Separate personal stylesheets from ready made libraries. First include the ready made and then the personal ones.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
+</<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
```
Comment (pkakelas) [inline index.html:4]: You should learn what this line does.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    <link rel="stylesheet" type="text/css "href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title></title>
```
Comment (pkakelas) [inline index.html:8]: Empty title

File: index.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  </head>
+
+  <body>
+    <div class="box">
```
Comment (pkakelas) [inline index.html:12]: Add proper class names

File: index.html [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+      <h2>Sign in</h2>
+      <form>
+        <div class = "form-group">
```
Comment (pkakelas) [inline index.html:15]: No space before and after `=`

File: index.html [hunk 7]
```diff
@@ -0,0 +1,41 @@
+</<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css "href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title></title>
+  </head>
+
```
Comment (pkakelas) [inline index.html:10]: Unneeded empty line

File: index.html [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+  <body>
+    <div class="box">
+      <h2>Sign in</h2>
+      <form>
+        <div class = "form-group">
+          <input type="user" class="form-control" id="exampleInputUser1" placeholder="User Name" >
```
Comment (pkakelas) [inline index.html:16]: Fix id naming. The id of this input should be clearly `username`. Also username is one word. Replace `User Name` with `username`

File: index.html [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+        <div class="form-group">
+          <input type="password" class="form-control" id="exampleInputPassword1" placeholder="Password" >
+        </div>
+        <div class = "form-group">
+          <button type="submit" class="btn btn-primary btn-lg btn-block">Sign in</button>
+        </div>
+        <p id="password">Forgot Password?</p>
+      </form>
+    </div>
+    <p id="sign-up">Not a member? <span>Sign up</span>
```
Comment (pkakelas) [inline index.html:27]: Where do you close the `p` tag

File: index.html [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+
```
Comment (pkakelas) [inline index.html:28]: Unneeded line

File: index.html [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+  <body>
+    <div class="box">
+      <h2>Sign in</h2>
+      <form>
+        <div class = "form-group">
+          <input type="user" class="form-control" id="exampleInputUser1" placeholder="User Name" >
+        </div>
+        <div class="form-group">
+          <input type="password" class="form-control" id="exampleInputPassword1" placeholder="Password" >
```
Comment (pkakelas) [inline index.html:19]: Fix id naming. Also personally I write placeholder texts in lowercase.

File: index.html [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+        </div>
+        <p id="password">Forgot Password?</p>
+      </form>
+    </div>
+    <p id="sign-up">Not a member? <span>Sign up</span>
+
+    <div class="box">
+      <div class="sticky-top"><h3 id="media">Log in with:</h3> </div>
+      <ul class="list">
+        <li><button type="button" class="btn btn-primary btn-lg btn-block" id="fb">Facebook     <i class="fa fa-facebook-square" style="font-size:24px"></i>
```
Comment (pkakelas) [inline index.html:32]: Separate each tag in new lines

File: index.html [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+        </button></li>
+        <li><button type="button" class="btn btn-primary btn-lg btn-block" id="tw">Twitter     <i class="fa fa-twitter" style="font-size:24px"></i>
```
Comment (pkakelas) [inline index.html:34]: Separate each tag in new lines.

File: index.html [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+        </button></li>
+        <li><button type="button" class="btn btn-primary btn-lg btn-block" id="google">Google+      <i class="fa fa-google-plus-official" style="font-size:24px"></i>
```
Comment (pkakelas) [inline index.html:36]: Separate each tag in new lines.

File: index.html [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+        </div>
+        <div class = "form-group">
+          <button type="submit" class="btn btn-primary btn-lg btn-block">Sign in</button>
+        </div>
+        <p id="password">Forgot Password?</p>
+      </form>
+    </div>
+    <p id="sign-up">Not a member? <span>Sign up</span>
+
+    <div class="box">
```
Comment (pkakelas) [inline index.html:29]: You never close this `div`

File: style.css [hunk 16]
```diff
@@ -0,0 +1,71 @@
+body {
+  background-image: url(peace.jpg);
```
Comment (pkakelas) [inline style.css:2]: Image paths should be inside quotes peace.jpg => './peace.jpg'

File: style.css [hunk 17]
```diff
@@ -0,0 +1,71 @@
...
+  background-repeat: repeat-y;
```
Comment (pkakelas) [inline style.css:3]: Not needed

File: style.css [hunk 18]
```diff
@@ -0,0 +1,71 @@
...
+  color: #00a4a2;
+  background: #222;
+}
+.box label {
+  color: #00a4a2;
+  margin-bottom: 0px;
+}
+.box button {
+  font-size: 16px;
+  text-size:16px;
```
Comment (pkakelas) [inline style.css:31]: space after `:`.

File: style.css [hunk 19]
```diff
@@ -0,0 +1,71 @@
...
+span {
+  cursor: pointer;
+  color: #00a5a0;
+}
+.list {
+  padding-left: 0px;
+}
+.list li{
+  list-style-type: none;
+  padding: 0 10% 3% 10%;
```
Comment (pkakelas) [inline style.css:54]: consistency in units. 0 => 0%

File: style.css [hunk 20]
```diff
@@ -0,0 +1,71 @@
...
+  cursor: pointer;
+  text-align: center;
+  font-size: 15px;
+  color: #00a4a2;
+}
+#sign-up {
+  text-align: center;
+  font-size: 15px;
+}
+span {
```
Comment (pkakelas) [inline style.css:45]: You will have many spans in the whole project. Use a class to do the same thing

File: index.html [hunk 21]
```diff
@@ -0,0 +1,41 @@
...
+        <div class = "form-group">
+          <input type="user" class="form-control" id="exampleInputUser1" placeholder="User Name" >
+        </div>
+        <div class="form-group">
+          <input type="password" class="form-control" id="exampleInputPassword1" placeholder="Password" >
+        </div>
+        <div class = "form-group">
+          <button type="submit" class="btn btn-primary btn-lg btn-block">Sign in</button>
+        </div>
+        <p id="password">Forgot Password?</p>
```
Comment (pkakelas) [inline index.html:24]: This should be a button or a `<a/>`

File: style.css [hunk 22]
```diff
@@ -0,0 +1,71 @@
+body {
+  background-image: url(peace.jpg);
+  background-repeat: repeat-y;
+  background-size: 70%;
+  position: fixed;
+  top: 10%;
+  left: 70%;
+}
+.box {
```
Comment (pkakelas) [inline style.css:9]: If you don't have more than one boxes then you should move with `id` here.

File: style.css [hunk 23]
```diff
@@ -0,0 +1,71 @@
...
+  width: 120%;
```
Comment (pkakelas) [inline style.css:10]: Don't add percentage more than 100%

File: index.html [hunk 24]
```diff
@@ -0,0 +1,41 @@
...
+      </form>
+    </div>
+    <p id="sign-up">Not a member? <span>Sign up</span>
+
+    <div class="box">
+      <div class="sticky-top"><h3 id="media">Log in with:</h3> </div>
```
Comment (pkakelas) [inline index.html:30]: Fix naming

File: index.html [hunk 25]
```diff
@@ -0,0 +1,41 @@
+</<!DOCTYPE html>
```
Comment (pkakelas) [inline index.html:1]: Syntax error

File: style.css [hunk 26]
```diff
@@ -0,0 +1,71 @@
...
+  border-radius: 2%p;
```
Comment (pkakelas) [inline style.css:11]: This unit is falsy. Also, since you've no borders defined then you probably don't need that.

File: style.css [hunk 27]
```diff
@@ -0,0 +1,71 @@
...
+  padding:24px 24px 16px 24px;
+  background: rgba(0,0,0,.9);
+  color: #fff;
+}
+.box h2 {
+  color: #00a4a2;
+  text-align: center;
```
Comment (pkakelas) [inline style.css:18]: Do that with bootstrap

File: style.css [hunk 28]
```diff
@@ -0,0 +1,71 @@
...
+  font-size: 30;
+}
+#exampleInputUser1 , #exampleInputPassword1{
```
Comment (pkakelas) [inline style.css:21]: Fix names.

File: index.html (hunk 25 above)
Comment (pkakelas) [inline index.html:1]: On front end changes a screenshot of your changes should come along with the PR (put it on the description).

File: index.html (hunk 1 above)
Comment (mpatsiou) [reply index.html:2]: Because atom put it automatically

File: index.html (hunk 3 above)
Comment (mpatsiou) [reply index.html:4]: ok

File: index.html (hunk 4 above)
Comment (mpatsiou) [reply index.html:8]: We are not sure about the name of the web app

File: style.css (hunk 17 above)
Comment (mpatsiou) [reply style.css:3]: Ιt is necessary because the photo appears many times

File: index.html (hunk 21 above)
Comment (mpatsiou) [reply index.html:24]: I am not sure about that

File: index.html (hunk 1 above)
Comment (pkakelas) [reply index.html]: This is not needed in western countries. You should remove that.

**Pull request closed without merging**